_posts/2019-05-24-server-rendering-responsively.md

```diff
@@ -34,25 +34,23 @@ medium-sized screens, and 100% for anything larger:
 </Box>
 ```
 
-The next example is based on actual code
-[from our codebase](https://github.com/artsy/reaction/blob/f3dabb884d616c5c42bbd47b1432dc2a9456b8ca/src/Apps/Search/Components/SearchResultsSkeleton/Header.tsx#L6).
+Another example:
 
 ```xml
-<Box height={100} marginBottom={30} marginTop={120} paddingRight={[0, 20]} paddingLeft={[0, 20]}>
-  <Box paddingLeft={[20, 0]}>
-    <Box marginBottom={40} background={color("black10")} width={260} height={20} />
+<Flex flexDirection={["column", "row"]}>
+  <Box px={40} background="black10">
```

Comment:
I think replacing `paddingLeft` with `px` here is not helping. People will either gloss over it or be left wondering.